esptfa_arima/Authentication/views.py

```
@@ -115,9 +134,67 @@ def create(self, request):
+            "total_students": Student.objects.count(),
+            "total_teachers": Teacher.objects.count(),
+            "total_sections": Section.objects.count(),
+            "total_subjects": Subject.objects.count(),
+            "total_documents": AnalysisDocument.objects.count(),
+        })
+
 class StudentViewSet(ModelViewSet):
+
     queryset = Student.objects.all()
     serializer_class = StudentSerializer
+    permission_classes = [permissions.IsAuthenticated, IsAdminUser]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)